glean-core/android/src/test/java/mozilla/telemetry/glean/net/HttpURLConnectionUploaderTest.kt

```diff
@@ -57,43 +55,7 @@ class HttpURLConnectionUploaderTest {
     }
 
     @Test
-    fun `Glean headers are correctly dispatched`() {
```

Comment:
Should we really remove this? It is checking that the Android object is reporting the same headers we're sending from Glean. Since this class is about testing the specific Android connection object, I'd leave it around

Reply:
Test still passes, so no, we probably shouldn't, adding it back.